### PR TITLE
MSEARCH-89 Add ability to specify language analyzer in configuration

### PR DESCRIPTION
**Pull request by pfilippovEPAM**
- Environment specific analyzer can be passed as part of language config entity
- Default analyzer for Korean language set to seunjeon_analyzer

### CODE REVIEW

File: src/main/resources/swagger.api/mod-search.yaml [hunk 1]
```diff
@@ -149,7 +149,7 @@ paths:
   /config/languages:
     post:
       operationId: createLanguageConfig
-      description: Save languages that will be used for analyzers
+      description: Update language configuration that will be used for analyzers
```
Comment (hjiebsco) [inline src/main/resources/swagger.api/mod-search.yaml:152]: For the description, do you mean `Create` or `Update`?

File: README.md [hunk 2]
```diff
@@ -36,7 +37,7 @@ The `code` here is an ISO-639-2/B three-letter code. Here is the list of pre-ins
 - heb
 - ita
 - jpn
-- kor
+- kor (default analyzer: seunjeon_analyzer, alternative for k8s and on-premise deployment - nori)
```
Comment (hjiebsco) [inline README.md:40]: Is `seunjeon_analyzer` available for k8s and on-premise by default? Or they have to call API to change to `nori` after deployment?

**Review by hjiebsco — APPROVED** [review]

**Review by MikhailFokanov — APPROVED** [review]
Looks good.

File: README.md (hunk 2 above)
Comment (pfilippovEPAM) [reply README.md:40]: `seunjeon_analyzer` is available only for AWS Elasticsearch.
'nori' or `seunjeon_analyzer` are not included in the default analyzers set in elasticsearch,  they should be installed in the Dockerfile or in the elasticsearch using command `elasticsearch-plugin install ${plugin}`.

**Review by hjiebsco — APPROVED** [review]

**Pull request merged**